### PR TITLE
Translation Update for Portuguese (Brazil)

### PR DESCRIPTION
**Pull request by euderdesousaa**
This pull request introduces a new translation for the project, with the goal of enhancing support for Brazilian users. The translation ensures that all key terms and phrases are accurately conveyed in Portuguese, while also improving the clarity of certain elements. This update aims to make the project more accessible and user-friendly for Portuguese-speaking players.

Thank you for considering this addition!

### CODE REVIEW

File: xmcl-keystone-ui/locales/pt-BR.yaml [hunk 1]
```diff
@@ -0,0 +1,1727 @@
...
+  enable: 'Pacotes de textura incluídos: {count}'
+  failed: Falhou
+  import: Importar texturepack
+  incompatible: Imcompatível
+  manage: Texturepack
+  manager: Gerenciador de Tarefas
+  nTaskRunning: '{count} tarefas em execução'
+  name: Tarefa | Tarefas
+  pause: Pausar
+  searchOnCurseforge: Pesquisar {nome} no curseforge
```
Comment (ci010) [inline xmcl-keystone-ui/locales/pt-BR.yaml:1546]: Guess this should be `{name}`?

**Review by ci010 — CHANGES_REQUESTED** [review]

File: xmcl-keystone-ui/locales/pt-BR.yaml (hunk 1 above)
Comment (euderdesousaa) [reply xmcl-keystone-ui/locales/pt-BR.yaml]: Damn, I'm sorry, that field I translated from the Russian language, the translator probably translated the English part into Portuguese.

**Comment by euderdesousaa** [comment]
Alteration

**Comment by ci010** [comment]
LGTM. Really appreciated!

**Review by ci010 — APPROVED** [review]

**Pull request merged**